collect-forms.rst

```diff
@@ -16,112 +20,118 @@ In order to fill out forms with survey participants, you must first load blank f
+       .. image:: /img/collect-forms/downloading-not-a-form.* 
+         :alt:
 
-.. _fill-blank-forms:
+       But you can't Fill Out a non-form.  
 
-Filling out forms
-===================
+       .. image:: /img/collect-forms/not-form-exception.* 
+         :alt:
 
-Once you have at least one form :ref:`loaded into ODK Collect <loading-forms-into-collect>`, you can fill out a form. 
+.. link to Google forms guide, once there is one
 
-- Select :guilabel:`Fill Blank Form` from the app home screen.
-- Select the form you would like to fill out from the form list.
-- Typically, you will :gesture:`swipe left` to advance forward through the question, and :gesture:`swipe right` to back up.
+.. _loading-forms-directly:
 
-  - You can switch from :gesture:`Swipe` navigation to Forward/Backward buttons in :menuselection:`⋮ -> General Settings -> User Interface`
-  - The |arrow| icon on the top right corner opens the jump menu. From the jump menu, you can go to any question or go to the beginning/ending of the form.
+Loading forms directly
+------------------------
 
-  .. |arrow| image:: /img/collect-forms/jumpicon.*
-             :alt: Opens the jump menu. 
+.. _loading-forms-with-adb:
 
-  |           
+Loading forms with ``adb``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-  .. image:: /img/collect-forms/jumpscreen.*
-    :alt: Screen with the arrow icon displayed in ODK Collect on an Android phone. 
-    :class: device-screen-vertical
-  
-  |
+You can load forms directly from a computer to your device via USB, using :doc:`Android Debug Bridge <collect-adb>`.
 
-  .. image:: /img/collect-forms/jumpmenu.*
-    :alt: Jump menu displayed in ODK Collect on an Android phone. 
-    :class: device-screen-vertical
+.. code-block:: none
 
-  |
-    
-  .. note::
-    Jump menu only shows the questions of a looped group once an actual record is created.   
-   
-  - Some questions will :ref:`auto-advance <autoadvance-widget>` after being answered.
-  - `Required questions <http://xlsform.org/#required>`_ will not allow you to advance unless answered.
+  $ adb push path/to/form.xml /sdcard/odk/forms/form.xml
 
-- To **remove a response**, :gesture:`Long Press` on the :term:`question label`. 
+Loading forms from device storage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  
+You can also download forms to your device via a web browser, and move them to the :file:`odk/forms/` directory, using the device's file manager (:menuselection:`Settings -> Storage & USB -> Explore`).
 
+1. Go to the Settings menu (:guilabel:`⚙`) on your device and find :menuselection:`Storage & USB`
 
-For a (mostly) complete guide to form question appearance, see :doc:`form-widgets`.
+   .. figure:: /img/collect-forms/device-settings-storage.* 
+     :alt: The Settings menu on an Android Device. The option *Storage and USB* is circled in red.
+  
+     The settings menu may look different on your device.
```

Comment:
This line can be added as a note.

Reply:
i disagree. it's a caption to the picture.

Reply:
Yes but maybe a caption should tell something about the picture and here this is just an additional information.

Reply:
I see @ankita240796's point here. It might read more as a caption if it were something like "The settings menu. It may look different on your device." I would tend to remove this entirely because really only the Nexus 5 running that version of Android will look like that or even have those options. I think that will be included in the issue @Divya063 is filing about the file explorer (or lack thereof).